api/agentpb/agent.proto

```diff
@@ -160,8 +160,14 @@ message CheckConnectionRequest {
 
 // CheckConnectionResponse is an AgentMessage containing a result of connection check.
 message CheckConnectionResponse {
+  // Stats contains various Service statistics.
+  message Stats {
+    // A number of tables. 0 means "unknown" (for compatibility), -1 means "there are really 0 tables".
```

Comment:
let's use extra boolean field instead of using -1

Reply:
Actually, it is not even possible: `SELECT COUNT(*) FROM information_schema.tables` always returns non-zero value – `information_schema.tables` is a table itself. 😆